java/core/src/main/java/com/microsoft/bond/StructBondType.java

```diff
@@ -13,12 +13,16 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
+import static com.microsoft.bond.Something.wrap;
```

Comment:
I am not a fan of this language feature, as it can easily make the code hard to read. Further, half of methods call Something.wrap explicitly and others rely on the imported method. Can you please make it consistent?